homework/english/Anastasia_Sheremet/lesson2/grammar.md

```diff
@@ -0,0 +1,40 @@
+Упражнения Past Simple и Present Perfect
+Упражнение 1. Put the verbs in brackets in the Past Simple or in the Present Perfect.
+
+    I have just had a nice pot of coffee. Would you like a cup?
+    I haven't seen Steve this morning yet.
+    Carol and I are old friends. I have known her since I was a freshman in high school.
+    Maria has had a lot of problems since she came to this country.
+    I went to Paris in 2003 and 2006.
+    A car came round the corner and I jumped out of the way.
+    Don’t throw the paper away because I have not read it yet.
+    Is Jim going to eat lunch with us today? — No. He has already eaten . He ate lunch an hour ago.
+    Since we started doing this exercise, we have completed some sentences.
+    I have never been to Italy.
+
+
+Упражнение 2. Put the verbs in brackets in the Past Simple or in the Present Perfect.
+
+    Have you ever worked in a shop?
+    I worked at my uncle’s shop when I was younger.
+    It’s the first time I have been on a ship.
+    Ann is looking for her key. She can’t find it. She has lost her key.
+    How many symphonies did Beethoven compose?
+    Look! Somebody has spilled ink on the notebook.
```

Comment:
spilt

Reply:
Both "spilt" and "spilled" are possible)))
https://english-verbs.ru/grammar/spill